source/base/Text.ooc

```diff
@@ -21,8 +21,20 @@ import math
 
 Text: cover {
 	_buffer: TextBuffer
-	count ::= this _buffer count
-	isEmpty ::= this count == 0
+	count: Int {
+		get {
+			result := this _buffer count
+			this free(Owner Receiver)
+			result
+		}
+	}
```

Comment:
Wouldn't work well with a setter though. (or?)
But with only a getter, yes, I like that too.